sync/sync-lib/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2022 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.duckduckgo.sync.lib">
+
+</manifest>
```

Comment:
We should remove the manifest and use `namespace` instead in the build.gradle, eg.
```groovy
android {
  namespace 'com.duckduckgo.sync.lib'
}
```

nit: the name of the library `com.duckduckgo.sync.lib` doesn't say much, specially the `lib` suffix.
Should we call it `com.duckduckgo.sync.crypto` for instance?